src/Run/CommandLine.cs

```diff
@@ -374,7 +374,7 @@ public static bool Parse(Action<CommandLineParser> parseBody, Action<CommandLine
         /// parsable types (values are comma separated without space).
         ///
         /// Qualifiers that are defined BEFORE any parameter sets apply to ALL parameter sets.  qualifiers that
-        /// are defined AFTER a parameter set will apply only the the parameter set that preceeds them.
+        /// are defined AFTER a parameter set will apply only the the parameter set that precedes them.
```

Comment:
Thanks for fixing the typo's but given that this is a fork of the other commandline.cs file they will still exist there :( Part of the reason forks suck.